demisto_sdk/commands/common/tools.py

```diff
@@ -198,7 +199,7 @@ def get_core_pack_list() -> list:
         return core_pack_list
     if not is_external_repository():
         core_pack_list = get_remote_file(
-            'Tests/Marketplace/core_packs_list.json', github_repo=GithubContentConfig.OFFICIAL_CONTENT_REPO_NAME
+            'Tests/Marketplace/core_packs_list.json', github_repo=GitContentConfig.OFFICIAL_CONTENT_REPO_NAME
```

Comment:
It supposed to be `github_repo`. When providing this argument, we assuming that the repo name is the github repo name, and getting the remote file from github (and not from the repo that we using). This is mainly used with `content` repo, when the content remote files are needed.
I didn't add support for getting remote files from other repos in gitlab, because we're not using it this way.